scripts/zones/Navukgo_Execution_Chamber/bcnms/omens.lua

```diff
@@ -0,0 +1,45 @@
+-----------------------------------
+-- Area: Navukgo Execution Chamber
+-- BCNM: BLU-AF2 Omens
+-----------------------------------
+require("scripts/globals/battlefield")
+require("scripts/globals/missions")
+local ID = require("scripts/zones/Navukgo_Execution_Chamber/IDs")
+----------------------------------------
+
+function onBattlefieldInitialise(battlefield)
+end
+
+function onBattlefieldTick(battlefield, tick)
+    dsp.battlefield.onBattlefieldTick(battlefield, tick)
+end
+
+function onBattlefieldRegister(player, battlefield)
+end
+
+function onBattlefieldEnter(player, battlefield)
+    local players = battlefield:getPlayers()
+    if #players ==  4 then GetMobByID(17039375):spawn(); end
```

Comment:
my total is 5. now what? where is your Altana now, Kain.

Reply:
Praise Altana in all Her might and glory. 🙏 

As the allied forces of Altana enter the battlefield, the script will make a check once each of them has enter the battlefield popping the enemies. In this case if five holy warriors join the battle on Her name, then 2 mobs will be in the area.

Now follow the sacred words sung on Altana's Hymn and hit that "merge pull request."

Reply:
gonna need to be made clearer still